cmd/monitor/monitor_apply.go

```diff
@@ -19,8 +19,8 @@ func MonitorApplyCommand(cli *di.Deps) *cobra.Command {
 	args := &ApplyArgs{}
 	cmd := &cobra.Command{
 		Use:   "apply",
-		Short: "Apply a monitor with STJ",
-		Long:  "Apply a monitor with StackState Templated JSON.",
+		Short: "Create or edit a monitor from STY",
```

Comment:
In theory you could still do a monitor apply with a stj as it uses under the hood the import functionality